infra/capi/cluster-capi-kind/cluster.yaml

```diff
@@ -30,7 +30,7 @@ spec:
         enabled: true
         enforce: baseline
         warn: restricted
-    version: v1.26.3
+    version: v1.31.0
```

Comment:
Would not just updating the kptfile help rather than updating the actual files every time?I could be wrong

Reply:
There are quite a few changes to the whole pkg so I don't think so. Changes to the CRDs etc.

Reply:
FYI It seems like we should be using something like the link below for changing image names and tags 
https://catalog.kpt.dev/set-image/v0.1/set-image-simple/

Reply:
I would somewhat agree but this is the base k8s version to be used in the workload clusters.
We could look to set this value dynamically later if needed but for now I am just updating the base version of each 3pp component.